src/parser.c

```diff
@@ -641,6 +641,17 @@ errr run_parser(struct file_parser *fp) {
  * The basic file parsing function
  */
 errr parse_file(struct parser *p, const char *filename) {
```

Comment:
I think this function should be called `parse_file_quit_not_found` and the more general one just `parse_file`.  At the moment when reading run_parse_lore() above, it looks the game should quit if the lore file is not found rather than the other way around.